_posts/2017-01-05-graduate-devops-blog.md

```diff
@@ -0,0 +1,22 @@
+---
+layout:     post
+title:      Taking a Chance - A Gradaute DevOps Blog
+date:       2017-01-05 12:00
+summary:    The transition of an engineering student into a DevOps graduate, through the Software Academy at Sky Betting and Gaming. 
+category:   Early Careers
+tags:       Gradautes, DevOps, Careers
+author:     david_sunter
+---
+
+Currently I’m a DevOps Graduate at Sky Betting and Gaming.  DevOps is not really an easy role to define, but from my brief experience I think that a DevOps engineer can be multi-skilled engineer, yet be a specialist in integration. This role is really interesting, and perfect for me as I get to be involved with all of the different aspects of the company, from testing, to development work and also the deployment of a product. What I am really trying to say is: if you like understanding the flow of information, solving problems by using analytical approaches or like to debug broken code – you should consider a role in DevOps.

```

Comment:
"I think that a DevOps engineer can be multi-skilled engineer"

Maybe "is" rather than "can be" would work better. "Can be" sounds a bit too speculative.

I might write "DevOps Engineers are usually multi-skilled engineers, making them a perfect for the kind of integration work the role requires"

or "multi-disciplinary"?